### ISSUE TITLE
Missing Entry: VPC Infrastructure Services

### ISSUE THREAD

**Issue by PKehnel** [ibm-cloud-docs/account#7]
The existing IAM Service: **VPC Infrastructure Services** is not documented in: https://cloud.ibm.com/docs/account?topic=account-iam-service-roles-actions#virtual-private-cloud

![Screenshot 2021-11-15 at 11 10 10](https://user-images.githubusercontent.com/23440847/141763227-b3a0dfcb-e703-4000-8a75-2cea2d5ae08b.png)

It contains uncommon service access attributes, like `Bare Metal Advances Network Operator`, for which we require API names.

**Comment by mekaufman** [comment]
@PKehnel Thank you for the feedback. We are aware of the current limitation in the linked documentation page, and we are working to address that gap. I'm hoping that this page can help you in the interim: https://cloud.ibm.com/docs/vpc?topic=vpc-resource-authorizations-required-for-api-and-cli-calls.

If that page doesn't have the answer that you're looking for, I encourage you to open an issue [here](https://github.com/ibm-cloud-docs/vpc/issues/new) so we can get the answers you need. Thanks again.

**Comment by PKehnel** [comment]
Kinda surprised that you close that issue before updating the documentation. 

And the attribute from my example: `Bare Metal Advances Network Operator` is not mentioned in your link. 

Maybe you can forward the issue to the correct team.

**Comment by mekaufman** [comment]
@PKehnel I have transferred this issue to the team who should be able to address this gap. See https://github.com/ibm-cloud-docs/vpc/issues/89 for tracking.

**Comment by mekaufman** [comment]
I did some testing, and it looks like the CRN values are the role IDs:

![image](https://user-images.githubusercontent.com/13648625/142224261-3dd2d57e-4dbb-4da2-8cd5-9d2cc91b51e9.png)

I assigned a user in my account a policy with the custom roles for the VPC Infrastructure Services service in the console. Then, I ran the CLI command for [reviewing assigned access by using the CLI](https://cloud.ibm.com/docs/account?topic=account-assign-access-resources&interface=cli&code=curl#review-your-access-cli) with the `--output json` parameter which gave me the IDs. I understand this is not ideal, and I'm looking into a way to improve this experience both in the console and our documentation. Thank you for reporting this issue.
